packages/app-desktop/commands/startExternalEditing.ts

```diff
@@ -3,7 +3,7 @@ import { _ } from '@joplin/lib/locale';
 import { stateUtils } from '@joplin/lib/reducer';
 import ExternalEditWatcher from '@joplin/lib/services/ExternalEditWatcher';
 import Note from '@joplin/lib/models/Note';
-const bridge = require('@electron/remote').require('./bridge').default;
+import bridge from '../services/bridge';
```

Comment:
Since both the main and renderer scripts are bundled, `require('@electron/remote').require('./bridge')` no longer results in the same instance of `bridge` as is used by the main process. The `bridge` must instead be transferred in a different way (see `/services/bridge.ts`).